cdc/entry/schema/snapshot.go

```diff
@@ -194,9 +200,36 @@ func NewEmptySnapshot(forceReplicate bool) *Snapshot {
 		forceReplicate:   forceReplicate,
 		currentTs:        0,
 	}
+
 	return &Snapshot{inner: inner, rwlock: new(sync.RWMutex)}
 }
 
+// InitConcurrentDDLTables imitate the creating table logic for concurrent DDL.
+// since v6.2.0, tables of concurrent DDL will be directly written to meta
```

Comment:
```suggestion
// Since v6.2.0, tables of concurrent DDL will be directly written to meta
```

Could you add more context? For example, where is "meta" located?